be/src/io/shared_buffered_input_stream.h

```diff
@@ -76,8 +75,10 @@ class SharedBufferedInputStream : public SeekableInputStream {
         int64_t ref_count;
         std::vector<uint8_t> buffer;
     };
+    Status _get_bytes(SharedBuffer& sb, const uint8_t** buffer, size_t offset, size_t* nbytes);
     StatusOr<SharedBuffer*> _find_shared_buffer(size_t offset, size_t count);
     std::shared_ptr<SeekableInputStream> _stream;
+    std::string _filename;
```

Comment:
I don't find any code using this variable

Reply:
No. I used this variable before, but I forget to clean. 

This variable perhaps will be used in future. 

I can remove them if you mind.